src/cswinrt/strings/WinRT.cs

```diff
@@ -491,7 +568,8 @@ public BaseActivationFactory(string typeNamespace, string typeFullName)
         public unsafe ObjectReference<I> _ActivateInstance<I>()
```

Comment:
Also double checking: I see you removed the `[UnconditionalSuppressMessage]` below. Is this one still needed or can it also be removed after the other changes you did?